pkg/cloudprovider/types.go

```diff
@@ -49,6 +53,27 @@ type NodeGroup interface {
 	Stabilized() (bool, string, error)
 }
 
+// Fleet represents a fleet request in the cloud provider,
+// all the instances in the fleet will have the same properties
+// number of instances can be controlled by setting the capacity
+type Fleet interface {
```

Comment:
I think we probably want to call this a Node and NodeClient (nodeClient.Create(...). Fleet is a very AWS specific concept, so if we're being vendor neutral (I think we should still try), we should avoid exposing it through the API.

Reply:
I had it as node before, but this interface lets user set the number of nodes to create, so I changed the name to fleet. May be I can change it to `nodes` or `nodeSet`

Reply:
Ack. How about Capacity?